server/api/products.js

```diff
@@ -0,0 +1,62 @@
+const Sequelize = require('sequelize');
+const router = require('express').Router()
+const { Product } = require('../../db')
+
+router.get('/', async function (req, res, next){
+	try {
+		const products = await Product.findAll()
+  		res.status(200).send(products)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}
+});
+
+router.get('/:productId', async function (req, res, next){
+	try {
+		const product = await Product.findOne({
+			where: {
+			  id: req.params.productId
+			},
+		})
+  		res.status(200).send(product)
+	} catch (error) {
+		console.log(error)
+		next(error)
+	}  	
+});
+
+router.post('/', function (req, res, next) {
+	Product.create(req.body)
+	.then(product => res.status(201).send(product))
+	.catch(next);
+  });
+
+  router.put('/:productId', (req, res, next) => {
+	const { name, description, inventory, price } = req.body;
+	const { productId } = req.params;
+	Product.findByPk(productId)
+	  .then(foundProduct => {
```

Comment:
what happens if there is no foundProduct? You should check and handle that case as well

Reply:
Incorporated